concepts/decorators/about.md

```diff
@@ -1,2 +1,210 @@
-#TODO: Add about for this concept.
+# About
 
+Decorators are a feature of the Python language that can modify or register a function.
+
+## How to use them
+
+Decorators are placed just above a function in the code like so:
+
+```py
+@decorator
+def function():
+    pass
+```
+
+Some decorators take arguments:
+
+```py

```

Comment:
```suggestion
```python

```

Reply:
we should spell out the language name for these code blocks.

Reply:
I assume this is for all code blocks?